docs/data/material/components/drawers/PersistentDrawerRight.tsx

```diff
@@ -101,7 +101,12 @@ export default function PersistentDrawerRight() {
           </IconButton>
         </Toolbar>
       </AppBar>
-      <Main open={open}>
+      <Main
+        open={open}
+        sx={{
+          position: 'relative',
+        }}
```

Comment:
Avoid using the `sx` prop. Instead, follow [the above comment](https://github.com/mui/material-ui/pull/38760#pullrequestreview-1642188029) and add it to the `styled` component.